cluster/gce/gci/health-monitor.sh

```diff
@@ -24,11 +24,22 @@ set -o pipefail
 
 # We simply kill the process when there is a failure. Another systemd service will
 # automatically restart the process.
-function docker_monitoring {
+function container_runtime_monitoring {
+  # Container runtime startup takes time. Make initial attempts before starting
+  # killing the container runtime.
+  local -r initial_attempts=5
+  local attempt=1
+  local -r crictl="${KUBE_HOME}/bin/crictl"
+  local -r container_runtime="${CONTAINER_RUNTIME_NAME:-"docker"}"
```

Comment:
nitpick: Remove the inner quotes. (You could remove all quotes here too, that would be fine.)

Reply:
It seems that google convention is to always quote `${}`.